pkg/raft/opts.go

```diff
@@ -46,13 +68,19 @@ var (
 )
 
 func init() {
-	flag.StringVar(&opts.nodeUrlStr, "nexusNodeUrl", "", "Url for the Nexus service to be started on this node (format: http://<local_node>:<port_num>)")
-	flag.StringVar(&opts.logDir, "nexusLogDir", "/tmp/logs", "Dir for storing RAFT logs")
-	flag.StringVar(&opts.snapDir, "nexusSnapDir", "/tmp/snap", "Dir for storing RAFT snapshots")
-	flag.StringVar(&opts.clusterUrl, "nexusClusterUrl", "", "Comma separated list of Nexus URLs of other nodes in the cluster")
-	flag.Int64Var(&replTimeoutInSecs, "nexusReplTimeout", 5, "Replication timeout in seconds")
-	flag.BoolVar(&opts.leaseBasedReads, "nexusLeaseBasedReads", true, "Perform reads using RAFT leader leases")
-	flag.StringVar(&opts.statsdAddr, "nexusStatsDAddr", "", "StatsD server address (host:port) for relaying various metrics")
+
+	flag.StringVar(&opts.nodeUrlStr, "nexus-node-url", "", "Url for the Nexus service to be started on this node (format: http://<local_node>:<port_num>)")
```

Comment:
Please update the README to reflect the changes to these params.

Reply:
Done